src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.Unix.cs

```diff
@@ -31,8 +32,31 @@ internal static string ParseFileName(string path, ZipVersionMadeByPlatform madeB
             }
             else
             {
+                PathInternal.CheckInvalidPathChars(path);
                 return Path.GetFileName(path);
             }
         }
+
+        internal static readonly char[] InvalidPathChars =
```

Comment:
Actually, we don't even need this anymore because of the below discussion. We no longer care if the path contains invalid characters.